great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -145,13 +149,30 @@ def _check_if_latest_version(self) -> None:
         checker = _VersionChecker(__version__)
         checker.check_if_using_latest_gx()
 
+    @override
+    def _init_analytics(self) -> None:
+        init_analytics(
+            user_id=self._get_cloud_user_id(),
+            data_context_id=uuid.UUID(self._data_context_id),
+            oss_id=self._get_oss_id(),
+            cloud_mode=True,
+        )
+
+    def _get_cloud_user_id(self) -> uuid.UUID:
```

Comment:
Should we cache this result? This can be [done with a decorator](https://docs.python.org/3/library/functools.html#functools.cache). It is fine not to but I don't think this should change for a context.

Reply:
We'll only ever call this on init once - since GX workflows aren't long-lived processes, does it make sense to cache?

Reply:
We can leave it as is and, if for some reason, we start hitting it multiple times we can update.